docs/content/deployment/guides/aws.mdx

```diff
@@ -1,19 +1,23 @@
 ---
-title: Deploying Dagster to AWS | Dagster
+title: Deploying Dagster to Amazon Web Services | Dagster
 description: To deploy Dagster to AWS, EC2 or ECS can host Dagit, RDS can store runs and events, and S3 can act as an IO manager.
 ---
 
-# Deploying Dagster to AWS
+# Deploying Dagster to Amazon Web Services
 
-This guide provides instructions for deploying Dagster on AWS. You can use EC2 or ECS to host Dagit and the Dagster Daemon, RDS to store runs and events, and S3 as an IO manager to store op inputs and outputs.
+This guide provides instructions for deploying Dagster on Amazon Web Services (AWS). You can use EC2 or ECS to host Dagit and the Dagster daemon, RDS to store runs and events, and S3 as an I/O manager to store op inputs and outputs.
```

Comment:
Maybe "You can use EC2 or ECS to host Dagit and the Dagster daemon" => You can use EC2 or ECS to host Dagster"? "Dagit and the Dagster daemon" leaves out grpc servers and runs

Reply:
"to host Dagster components" maybe?